apps/roam/src/components/results-view/ResultsView.tsx

```diff
@@ -1361,7 +1361,7 @@ const ResultsView: ResultsViewComponent = ({
                 <Kanban
                   data={allProcessedResults}
                   layout={layout}
-                  onQuery={() => onRefresh(true)}
+                  onQuery={() => onRefresh()}
```

Comment:
Could you add some context as to why this change was made, please?  Kanban quite seems unrelated to discourse context UI.

Reply:
That is because I removed the loadInBackground flag, so I removed the one call instance which seemed to be using it. So it depends on what we decide to do with that flag, above.